crates/biome_cli/tests/snapshots/main_cases_vcs_ignored_files/ignores_file_inside_directory.snap

```diff
@@ -30,6 +29,12 @@ internalError/io ━━━━━━━━━━━━━━━━━━━━━
 
   × No files were processed in the specified paths.
   
+  i Check your biome.json to ensure the paths are not ignored by the configuration.
+  
+  i There paths were provided but ignored:
+  
+  - /private<TEMP_DIR>/ignores_file_inside_directory
```

Comment:
It seems that the regex we use to redact the temporary directory isn't working properly

Reply:
Yeah, this is because I'm using macOS 😓 
There was a very simillar issue before: https://github.com/biomejs/biome/pull/5227